doc/changes.md

```diff
@@ -1,5 +1,113 @@
 # Changelog
 
+## Release v0.14.0 (2024/12/31)
+
+[![Digital Object Identifier for PyGMT v0.14.0](https://zenodo.org/badge/DOI/10.5281/zenodo.14535921.svg)](https://doi.org/10.5281/zenodo.14535921)
+
+### Highlights
+
+* 🎉 Fourteenth minor release of PyGMT 🎉
+* Bump minimum supported version to GMT>=6.4.0 ([#3450](https://github.com/GenericMappingTools/pygmt/pull/3450))
```

Comment:
```suggestion
* 🎉 Fourteenth minor release of PyGMT 🎉
* Bump minimum supported version to GMT>=6.4.0 ([#3450](https://github.com/GenericMappingTools/pygmt/pull/3450))
* Two new plotting methods and five new functions to access more GMT remote datasets
```

Reply:
I think we should move Python 3.13 under maintenance. Previous PyGMT (v0.13.0 and lower) versions should work with Python 3.13 also, it's just that we are testing it on CI now.

Reply:
OK, moved back to maintainence in e69fb3f2906bd43c7509ed3c59585e953d5181d5